public/styles.css

```diff
@@ -119,6 +119,7 @@ table {
 
 .mobile-content {
     padding: .5rem;
+    min-width: 350px;
```

Comment:
Yep, no good. :)
There are mobile devices with smaller widths and this will break them.